js/LocationButtons.js

```diff
@@ -0,0 +1,25 @@
+export default class LocationButtons extends HTMLElement {
+    constructor() {
+        super();
+        this.locationValue;
+        this.locationButtons = this.querySelectorAll('button');
+    }
+
+    connectedCallback() {
+        this.setupEventListener();   
+    }
+
+    setupEventListener() {
+        this.locationButtons.forEach( (button) => {
```

Comment:
Wenn der einzelne Button (statt alle Buttons) eine Komponente wäre, dann müsstest du a) den Button im Constructor nicht selecten und b) hier den Loop nicht machen. `this.addEventListener(() => {})` würde genügen.